admin-policy.md

```diff
@@ -0,0 +1,78 @@
+- Any changes to a PR before merging should ideally be made by the author of the PR. Pull requests by others to PRs are generally not allowed. The exception to this is when permission is given by the PR author due to the author being unable to make the discussed changes at that time.
+- Note that commits should never be removed from PRs (i.e. through rebasing/squashing), as we would like to preserve the entire history of the PR.
+
+## Management of Administration
+
+- We will increase the number of admins per term as the current admin team sees fit. A general indicator of how many admins are needed is to see how long it takes posts to be moderated. If all bad posts are caught within some small amount of time (i.e. 30 minutes) of posting, then we have enough admins
+- The length of a term for an admin will be 3 months
+- An admin can serve for a maximum of two consecutive terms, but there is no maximum limit on the number of terms an admin can serve for
+- All admin activity will be public. In the case of bans, see the ban policy later in this document. For larger, more official meetings, either the entire chat log will be posted (possibly as a GitHub commit or a gist), or paraphrased meeting minutes. Small, one-off conversations/meetings do not need to be public. Ideally, these meetings should occur in the ##hx-admin IRC channel on Freenode.
+- It is *highly* recommended that admins idle on ##hx-admin so that the community can easily have a short discussion or ask questions directly to the admins in a public area, without polluting the main HX group. HX Meta is also alright for this purpose, but an IRC conversation can often provide immediate feedback.
+
+## Election of Admins
+
+- The current admin team will determine how many admins are elected for the next term. 
+- 14 days before the end of a term, the web interface will allow candidates to mark themselves as running. 
+  - Candidates must be nominated by three group members via the web interface (after they have marked themselves as running) to be put on the ballot. Each group member can nominate one member per term.
+- 7 days before the end of a term, the web interface will allow group members to vote for a candidate. 
+- Votes will stop being accepted after 11:59PM GMT on the day the term ends. The current admins will use the voting result to add/remove themselves, if needed.
+- The voting system will publish a freely downloadable list of UUIDs and their vote after votes stop being accepted.
+- The election dates for each year will be January 4th, April 4th, July 4th, and October 4th.
+
+### Voting
+
+- Votes are counted via a voting web interface.
+  - In the event the web interface becomes inaccessible while voting is in progress, the voting period and moderation term may be extended to the nearest hour based on the outage length.
+- The group will have one official maintainer for the voting web interface.
```

Comment:
@iangcarroll I heard you like human error? Although, it will be revisioned... I still think it will be a pain to search. I guess I can live with that.